src/nwnsdk/postgres/postgres_client.py

```diff
@@ -62,14 +59,46 @@ def store_job_result(self, job_id: uuid4, new_logs: str, new_status: JobStatus,
             stmnt = (
                 update(Job)
                 .where(Job.job_id == job_id)
-                .values(status=new_status.value, logs=new_logs, output_esdl=output_esdl, stopped_at=datetime.now())
+                .values(status=new_status, logs=new_logs, output_esdl=output_esdl, stopped_at=datetime.now())
             )
             session.execute(stmnt)
 
+    def get_job_status(self, job_id: uuid4) -> JobStatus:
+        LOGGER.debug("Retrieving job status for job %s", job_id)
+        with session_scope() as session:
+            stmnt = select(Job).options(load_only(Job.status)).where(Job.job_id == job_id)
+            job: Job = session.scalar(stmnt)
+            session.expunge(job)
+        return job.status
+
     def get_job(self, job_id: uuid4) -> Job:
-        session: Session
         LOGGER.debug("Retrieving job data for job %s", job_id)
         with session_scope() as session:
             stmnt = select(Job).where(Job.job_id == job_id)
             job: Job = session.scalar(stmnt)
+            session.expunge(job)
         return job
+
+    def get_jobs(self, job_ids: list[uuid4]) -> Job:
```

Comment:
`list[uuid4]` is not python3.8 compatible. We should use `List[uuid4]` and `from typing import List`